galaxy/accounts/models.py

```diff
@@ -90,29 +93,37 @@ class Meta:
     def __str__(self):
         return self.username
 
+    # FIXME: replace with django.core.urlresolvers.reverse(..)
     def get_absolute_url(self):
         return "/users/%s/" % urlquote(self.username)
 
+    # FIXME: This method looks unused
     def get_full_name(self):
         """
         Returns the first_name plus the last_name, with a space in between.
         """
         full_name = self.full_name
         return full_name.strip()
 
+    # FIXME: This method looks unused
     def get_short_name(self):
         """Returns the short name for the user."""
         return self.short_name.strip()
 
+    # FIXME: This method looks unused
     def get_num_roles(self):
         return self.roles.filter(active=True, is_valid=True).count()
 
+    _send_email = staticmethod(send_mail)  # for testing purposes
+
+    # FIXME: This method looks unused
     def email_user(self, subject, message, from_email=None):
         """
         Sends an email to this User.
         """
-        send_mail(subject, message, from_email, [self.email])
+        self._send_email(subject, message, from_email, [self.email])
 
+    # FIXME: This method looks unused
```

Comment:
Looks like this method was left by predecessors. I thought I removed it.
Definitely not needed.

Reply:
Could you please walk through all these labels and comment, what's really needed and what's not.